loadtest/control/simulcontroller/utils.go

```diff
@@ -60,11 +60,23 @@ func splitName(name string) (string, string) {
 	return prefix, typed
 }
 
+func getCutoff(prefix, typed string, altRand *rand.Rand) int {
+	cutoff := len(prefix) + 2
+	if len(typed)/2 > 0 {
+		if altRand != nil {
+			cutoff += altRand.Intn(len(typed) / 2)
+		} else {
+			cutoff += rand.Intn(len(typed) / 2)
+		}
+	}
+	return cutoff
```

Comment:
nit: maybe we can write this a little bit more expressive like:
```suggestion
	switch {
	case len(typed)/2 > 0 && altRand != nil:
		return cutoff + altRand.Intn(len(typed)/2)
	case len(typed)/2 > 0:
		return cutoff + rand.Intn(len(typed)/2)
	default:
		return cutoff
	}
```

Reply:
I'm 2/5 on having a single return. Here I think the function is short enough that the switch is clearer, so applied!